src/components/PostImage.tsx

```diff
@@ -0,0 +1,11 @@
+import Image from "next/image"
+import demo from "../../public/profile.jpg";
+
+
+const PostImage = ({image}:{image:string }) => {
+  return (
+    <Image width={800} className="m-2" src={demo} alt="photo" />
+  )
+}
```

Comment:
does this have to be a separate component?

Reply:
resolved